📝 Customers may see a brief description of the underlying value proposition for Compensated

zspencer opened the pull request:
Part of #29

zspencer committed:
Rough draft of a home page for compensated



Part of #29

cjerozal commented on www/index.md:
```diff
@@ -0,0 +1,26 @@
+
+### Expand Your Sales Channels
+You started with a website, and now you have an iOS app.
+And oh yea, there’s that old Gumroad storefront you were experimenting with that has some recurring payments set up.
```
I would make a slight adjustment to acknowledge these are only possible cases:

> Maybe you started with a website, and now you have an iOS app. And oh yeah, maybe there's that old Gumroad storefront you were experimenting with that has some recurring payments set up.

zspencer replied:
Thank you! I like that! (Probably will adjust Friday)

cjerozal commented on www/index.md:
```diff
@@ -0,0 +1,26 @@
+You started with a website, and now you have an iOS app.
+And oh yea, there’s that old Gumroad storefront you were experimenting with that has some recurring payments set up.
+
+No matter how you get (or got!) paid, Compensated standardizes the different payment processers data structures and event types so you can add sales channels without losing momentum on your customer facing features.
```
processors'

Edit: 
Actually... I want to comment on more than just the spelling and missing possessive. I like the message but it feels a bit wordy. How about?:

> No matter how you get paid, Compensated standardizes the different payment processors' data structures and event types. So you can make money while focusing on your product, rather than on obscure payment details.

zspencer replied:
I dig it!

cjerozal commented on www/index.md:
```diff
@@ -0,0 +1,26 @@
+No matter how you get (or got!) paid, Compensated standardizes the different payment processers data structures and event types so you can add sales channels without losing momentum on your customer facing features.
+
+### Prevent Vendor Lock-In
+Tired of your current payment providers fee structure?
```
providers'

cjerozal commented on www/index.md:
```diff
@@ -0,0 +1,26 @@
+Compensated helps you safely transition your customers to new payment processors without requiring system rewrites.
+
+### Read the Source, Luke
+All our source code is publicly available.
```
I haven't really worked under this model for a paid product. Couldn't we just give access to the paying customers at first?

zspencer replied:
An excellent question! I think we _could_, but it would mean we would want a source distribution mechanism other than "push to publicly readable git repository" and I don't have the time or expertise to set that up. 

I'd personally prefer to rely on existing copyright laws. Someone who can read a book from the library doesn't have the right to copy the book or re-sell the copies to their friends.  If someone violates the terms of the license, we can send them a cease and desist.

If they refuse, we could pursue further legal action commiserate with our economic ability to do so. Having a public record of source code contributions with timestamps gives us a public record of the intellectual property being owned by us and makes the case pretty clear.

cjerozal replied:
Yeah, managing the source release could get annoying. I guess the case I'm most worried about is people who might use this library or might decide to roll their own similar thing. If our source is out for anyone to see, people have a lot of sample code to help them understand how the systems work. I think the value of the product will probably be more in that understanding than in the exact code itself. If we want to put the source out there, I might prefer actual open source plus expert consulting that we focus on. But also, unfamiliar things are scarier so I could be worrying about a non-issue.

zspencer replied:
I would love to license this as an OSI approved license once they pass something that allows us to not grant a license to organizations that violate human rights. 

And yes, I agree that "open source + hey if you want support + here's a hosted version" is the economic model that would work best and is pretty familiar.

I could also see an opportunity for a "compensated-pro" that is similar to how Sidekiq provides a pro edition of the library

cjerozal commented on www/index.md:
```diff
@@ -0,0 +1,26 @@
+
+### Read the Source, Luke
+All our source code is publicly available.
+We build in the open, and grant usage rights freely to any and all individuals. 
```
...Except the individuals described below

zspencer replied:
I'm not sure what you mean here?

cjerozal replied:
"We ... grant usage rights freely to any and all individuals." But then we say "we have legal standing to refuse to grant usage rights to organizations that violate human rights". (Which is cool.) So it's not to "any and all" individuals. And "freely" is a bit confusing since it's a paid product, though technically used in a different sense here. I guess I just don't get what this part is trying to communicate.

zspencer replied:
Ah, so the thing to keep in mind is that an individual acting on behalf of an organization is distinct from an individual acting on behalf of oneself. Someone who say, installs the gem and pushes it to production for a service being operated by the red cross, is an individual acting on behalf of an organization. That person _may_ choose to claim that the software they are using the gem in is operated by them as an individual, but it would be unlikely to hold water.

I am definitely interested in hearing how you would phrase this to disambiguate between the two things; but it's not a paradox; I swear!

cjerozal replied:
Ah I see... So we don't ever want to prevent an _individual_ from using, even a really bad (for some definition we determine) individual? And is a sole proprietor still also an individual? 

Edit: One more Q. Is the plan to charge organizations for use but not individuals?

cjerozal reviewed (approved):
Nice page!

user512 reviewed (approved):
Looks great

zspencer committed:
Revise landing page copy with Cheryl

We wanted to focus the message down to the C/VP/Director level
engineering person who can make the purchase decision; as well as make
it more clear that it's an early stage product.
